third_party/move/move-compiler-v2/src/experiments.rs

```diff
@@ -129,9 +129,19 @@ pub static EXPERIMENTS: Lazy<BTreeMap<String, Experiment>> = Lazy::new(|| {
         },
         Experiment {
             name: Experiment::OPTIMIZE.to_string(),
-            description: "Turns on or off a group of optimizations".to_string(),
+            description: "Turns on standard group of optimizations".to_string(),
             default: Given(true),
         },
+        Experiment {
+            name: Experiment::OPTIMIZE_NONE.to_string(),
```

Comment:
I did not expect this to become an experiment flag, but rather we define

```
const OPTIMIZE_NONE_EXPERIMENTS = vec![...]
````

Why make an extra experiment out of this? Also can't see it used.

Reply:
Fine.

Reply:
I'm not quite sure what you mean with "fine". If you resolved a comment please say "done". (And hit the resolve button to on 2nd review your intention of resolving is clear. Comment author can still reopen if needed.)

Reply:
I meant, "done"..